doc/smart-switch/reboot/reboot-hld.md

```diff
@@ -125,30 +126,30 @@ will be set to false in the RebootStatusResponse. Until the services are termina
 
 reboot(self, reboot_type):
 ```
-Define new reboot_type as MODULE_REBOOT_DPU for DPU only reboot and MODULE_REBOOT_SMARTSWITCH for entire switch reboot.
+Define new reboot_types as MODULE_REBOOT_DPU for DPU only reboot and MODULE_REBOOT_SMARTSWITCH for rebooting the DPU when entire switch is undergoing for a reboot.
 ```
 
 This API is defined in [smartswitch-pmon.md](https://github.com/sonic-net/SONiC/blob/master/doc/smart-switch/pmon/smartswitch-pmon.md#:~:text=reboot(self%2C%20reboot_type)%3A)
 
 ### ModuleBase Class new APIs ###
 
-pci_detach(self. module_name):
+pci_detach(self):
 ```
 Detaches the DPU PCI device from the NPU. In the case of non-smart-switch chassis, no action is taken.
 
 Returns:
     True
 ```
 
-pci_reattach(self, module_name):
+pci_reattach(self):
```

Comment:
@vvolam nit pci_rescan() ?

Reply:
@prgeor done. Thanks!

Reply:
> @vvolam nit pci_rescan() ?

@prgeor, Though I modified, wanted to rename back as pci_reattach, as pci_rescan is Linux kernel standard function name. To avoid confusion and conflicts better to avoid same names.